packages/rum-core/karma.bench.conf.js

```diff
@@ -35,7 +38,7 @@ module.exports = function(config) {
     JSON.stringify(customConfig, null, 2)
   )
   config.set(customConfig)
-  const specPattern = 'test/**/*.bench.js'
+  const specPattern = `${BENCHMARKS_DIR}/*.bench.js`
```

Comment:
`${BENCHMARKS_DIR}/**/*.bench.js` would be better since it allows us to restructure the benchmarks directory.

Reply:
Sure,